.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       run: cargo clippy -- -D warnings
 
     - name: Build | Rustfmt
-      run: cargo fmt --all -- --check
+      run: cargo fmt -- --check
```

Comment:
Why are these changes here rather than a separate PR?

Reply:
The examples didn't compile yet because of bevy_pancam not being updated yet